src/V3LinkDotIfaceCapture.h

```diff
@@ -0,0 +1,70 @@
+
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Interface typedef capture helper: stores (refp, typedefp,
+//    cellp, owners, pendingClone) so LinkDot can rebind refs when symbol
```

Comment:
First description line should be able to be read "stand alone"

```suggestion
// DESCRIPTION: Interface typedef capture helper
//    stores (refp, typedefp,
//    cellp, owners, pendingClone) so LinkDot can rebind refs when symbol
```